src/util/searcher-utils.js

```diff
@@ -21,3 +21,16 @@ export const applyNumberCircle = (number) => (
     {number}
   </div>
 );
+
+export const LOC_LEVELS = 4;
```

Comment:
Is this hardcoded value (max 4 level of locations)? Is this possible to be configurable? It's not the request of change but rather a question. I remember there were many discussions about this topic (how to configure level of locations to be flexible for specific implementations).